### ISSUE TITLE
Is it possible to suppress some kind of message?

### ISSUE THREAD

**Issue by 00sapo** [julia-vscode/LanguageServer.jl#959]

**Comment by davidanthoff** [comment]
Yes, there are plenty of options, take a look at the settings in the VS Code extension.

**Comment by 00sapo** [comment]
I use vim. Usually, such options are set from within the code with comment directives or from a standalone configuration file. Is this possible with LanguageServer?

**Comment by 00sapo** [comment]
In vim, we can set some options ([here](https://github.com/neovim/nvim-lspconfig/blob/master/CONFIG.md#julials) a list) but it seems not possible su suppress errors on specific lines via comments.

**Comment by davidanthoff** [comment]
Yes, that is true, and would actually be a useful feature! I'm changing the title here to track that, ok?
